src/test/java/gov/nasa/cumulus/CnmToGranuleHandlerTest.java

```diff
@@ -80,4 +80,44 @@ public void testPerformFunction() {
             e.printStackTrace();
         }
     }
+	
+	/**
+	 * Test that PerformFunction functions as expected when provided a CNM with filegroups
+	 */
+	public void testPerformFunctionFilegroups() throws Exception {
+		ClassLoader classLoader = getClass().getClassLoader();
+		File inputJsonFile = new File(classLoader.getResource("inputFilegroups.json").getFile());
+		File expectedJsonFile = new File(classLoader.getResource("expectedFilegroups.json").getFile());
+		
+		String input = new String(Files.readAllBytes(inputJsonFile.toPath()));
+		String expected = new String(Files.readAllBytes(expectedJsonFile.toPath()));
+		JsonObject expectedJson = new JsonParser().parse(expected).getAsJsonObject();
+		
+		CnmToGranuleHandler cnmToGranuleHandler = new CnmToGranuleHandler();
+		String output = cnmToGranuleHandler.PerformFunction(input, null);
+		JsonObject outputJson = new JsonParser().parse(output).getAsJsonObject();
+		
+		assert(expectedJson.equals(outputJson.getAsJsonObject("output")));
+    }
+	
+	/**
+	 * Test that PerformFunction functions as expected when provided a CNM with filegroups AND files
+	 *
+	 * The result should be the same as testPerformFunctionFilegroups
+	 */
+	public void testPerformFunctionFilegroupsAndFiles() throws Exception {
```

Comment:
This test case isn't needed since per CNM schema there will either be files or filegroups and not both https://github.com/podaac/cloud-notification-message-schema/blob/master/cumulus_sns_schema.json#L190-L193

Reply:
This test case has been removed